python/cudf/cudf/tests/test_ops.py

```diff
@@ -84,6 +85,12 @@ def test_acos(dtype, test_df):
     math_op_test(dtype, np.arccos, test_df=test_df)
 
 
+@pytest.mark.parametrize("dtype", int_type)
+@pytest.mark.parametrize("test_df", [False, True])
+def test_acos_integer(dtype, test_df):
+    math_op_test(dtype, np.arccos, test_df=test_df)
```

Comment:
```suggestion
    math_op_test(dtype, np.arccos, test_df=test_df, check_dtype=False)
```

We can parametrize `check_dtype` in `math_op_test`. Ensure we add it as a parameter with default value as `True` in `math_op_test` as we would want to check types incase of other ops..

Reply:
Thank you, this sounds good!